corehq/feature_previews.py

```diff
@@ -157,7 +158,8 @@ def can_enable_callcenter(request):
         'From here they can do things like monitor workers with performance issues, '
         'update their case with possible reasons for poor performance, '
         'and offer guidance towards solutions.'),
-    help_link='https://help.commcarehq.org/display/commcarepublic/How+to+set+up+a+Supervisor-Call+Center+Application',
+    help_link=('https://help.commcarehq.org/display/'
+               'commcarepublic/How+to+set+up+a+Supervisor-Call+Center+Application'),
```

Comment:
This link needs to be updated or removed too, right? The page it redirects to is archived, so my bet is "removed".